src/events/RecurrentPaymentStoppedEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Crm\PaymentsModule\Events;
+
+use League\Event\AbstractEvent;
+use Nette\Database\Table\ActiveRow;
+
+class RecurrentPaymentStoppedEvent extends AbstractEvent
```

Comment:
You didn't use this event for `SYSTEM_STOP` event which is good because when recurrent payment fails, we already emit `RecurrentPaymentFailEvent`.

What bothers me a bit is name of this event, because someone could expect that it includes all "stop" events: user/admin/system.

- Do you want to emit this also for SYSTEM_STOP? It would be in the `RecurrentPaymentProcessor`?
- If not, can you think of any better name suggesting it's only user/admin stops? :)

Reply:
Yes, I saw there was already a `RecurrentPaymentFailEvent`, I can use that for `SYSTEM_STOP`. Maybe 2 separate events would be better (`RecurrentPaymentStoppedByUser` and `RecurrentPaymentStoppedByAdmin`), if they should be handled in the same way, we can easily register the same handler for both types..

Reply:
Sounds good to me! Will you make the change?

Reply:
done